modules/shapes/widgets/text-path.php

```diff
@@ -344,13 +344,16 @@ protected function register_style_tab() {
 			[
 				'label' => esc_html__( 'Word Spacing', 'elementor' ),
 				'type' => Controls_Manager::SLIDER,
-				'size_units' => [ 'px' ],
+				'size_units' => [ 'px', 'em' ],
 				'range' => [
 					'px' => [
 						'min' => -20,
 						'max' => 20,
 						'step' => 1,
 					],
+					'em' => [
+						'step' => 0.1,
```

Comment:
```suggestion
						'min' => -1,
						'max' => 1,
						'step' => 0.1,
```

I've tested the PR. Looks good!

But in this scope, without the `min` / `max` it's not usable.

Please add the min/max values. I've tested with other values, but between `-1` to `1` feels like they are the best.

Reply:
Done.